### DISCUSSION TITLE
Goodwe GW5000N-EH inverter support

### DISCUSSION THREAD

**Discussion by chuotde** [ai-republic/bms-to-inverter#86] [Q&A]
Hi ai-republic,

I can find the CAN bus specification for GW6000-EH at https://secondlifestorage.com/index.php?attachments/can_protocol_hv1_310321-pdf.24481/. I believe that should be compatible with my inverter GW5000N-EH. Do you have any plan to implement that Goodwe CAN bus specification?

Thank you.

**Comment by ai-republic** [comment]
Hi @chuotde,
I guess I can implement that during the holidays. Will keep you updated 😄

**Comment by ai-republic** [comment]
@chuotde I added the binding for the Goodwe HV CAN _GOODWE_HV_CAN_. Please download the latest Configurator app and set the log level to _debug_ so that we can see if everything works fine.

**Reply by chuotde** [reply]
Thanks for your prompt support. Unfortunately, the battery modules I've just got is LV model (ELPM482-00005) so I need to change the inverter to LV model (connecting those modules in series to make them HV won't be a good choice in this case, I guess) . That's why I didn't have any chance to test your new binding. By the way, your software is still very useful to me and others. Thank you very much for your work.
